profile-widget/diveeventitem.cpp

```diff
@@ -194,7 +194,7 @@ void DiveEventItem::setupToolTipString(struct gasmix *lastgasmix)
 		}
 		*lastgasmix = *mix;
 	} else if (same_string(internalEvent->name, "modechange")) {
-		name += tr(": %1").arg(divemode_text[internalEvent->value]);
+		name += QString(": %1").arg(tr(divemode_text[internalEvent->value]));
```

Comment:
I think this does not work as the string to be translated has to be known at compile time.